packages/psss/src/containers/Panels/AlertsPanel.js

```diff
@@ -70,60 +69,72 @@ const menuOptions = [
 
 const TabsContext = React.createContext(null);
 
-export const AlertsPanel = React.memo(({ isOpen, handleClose }) => {
-  const [isModalOpen, setIsModalOpen] = useState(false);
-  const [activeIndex, setActiveIndex] = useState(0);
-  const sitesState = useFetch(getAffectedSites);
-  const notesState = useFetch(getAlertsMessages);
-  const activityState = useFetch(getActivityFeed);
-  const { countryCode } = useParams();
-  const countryName = useSelector(state => getCountryName(state, countryCode));
+export const AlertsPanel = React.memo(
+  ({ countryCode, period, syndromeName, isOpen, handleClose }) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const [activeIndex, setActiveIndex] = useState(0);
+    const sitesState = useFetch(getAffectedSites);
+    const notesState = useFetch(getAlertsMessages);
```

Comment:
I assume that none of the functionality of the tabs in the panel is supposed to work yet. As you can see the data for the tabs is just stubbed out using the fake api data. Also that useFetchHook is from a time before react-query was added to the project so it should be considered legacy and replaced with a react-query strategy when the real data is available.

Reply:
This is my assumption too!